src/extract_topic.py

```diff
@@ -0,0 +1,65 @@
+import nltk
+import glob
+from nltk.stem import WordNetLemmatizer
+from sklearn.feature_extraction.text import TfidfVectorizer, CountVectorizer
+from sklearn.decomposition import NMF, LatentDirichletAllocation
+
+
+
+
+def generate_text_li(text_dir):
+    text_li = glob.glob(text_dir+'/*.txt')
+    text_data = []
+    for file_name in text_li:
+        with open(file_name) as f:
+            #data = f.readlines()
+            data = " ".join(line.strip() for line in f)
+            text_data.append(data)
+    return text_data
+
+
+
+class LemmaCountVectorizer(CountVectorizer):
+    def build_analyzer(self):
+        lemm = WordNetLemmatizer()
+        analyzer = super(LemmaCountVectorizer, self).build_analyzer()
+        return lambda doc: (lemm.lemmatize(w) for w in analyzer(doc))
+
+
+def process_text2vec(text):
+
+    
+    tf_vectorizer = LemmaCountVectorizer(max_df=0.95,
```

Comment:
what's `tf` here? Use another name if it does not refer to tensforflow for better readability.